package.json

```
@@ -807,6 +908,62 @@
                     "default": "test/runtests.jl",
                     "description": "A workspace relative path to a Julia file that contains the tests that should be run for live testing.",
                     "scope": "window"
+                },
+                "julia.debuggerDefaultCompiled": {
+                    "type": "array",
+                    "default": [
+                        "Base.",
+                        "-Base.!",
+                        "-Base.all",
+                        "-Base.all!",
+                        "-Base.any",
+                        "-Base.any!",
+                        "-Base.atexit",
+                        "-Base.atreplinit",
+                        "-Base.cd",
+                        "-Base.disable_sigint",
+                        "-Base.filter",
+                        "-Base.filter!",
+                        "-Base.findall",
+                        "-Base.findfirst",
+                        "-Base.findlast",
+                        "-Base.findnext",
+                        "-Base.findprev",
```

For comprehendsion to work we need to do:
```suggestion
                        "-Base.findprev",
                        "-Base.Generator",
```
right?

---

Yes, but unfortunately that's not enough.
`[f(x) for x in x]` ends up calling 
```
(::Type{Base.Generator})(f::F, iter::I) where {I, F} in Base at generator.jl:32
```
which we catch fine with the above definition, but that then calls the parametric constructor
```
(::Type{Base.Generator{I,F}})(f, iter) where {I, F} in Base at generator.jl:32
```
which isn't included in `methods(Base.Generator)`. So I'm not sure where to get that from...

---

Was just missing an `unwrap_unionall` here.